examples/graphql-caching-global/graphql_caching_global.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/graphql;
+
+// The `cacheConfig` in `graphql:ServiceConfig` annotation is used to configure the cache for the GraphQL service.
```

Comment:
IIRC, the character limit is 100 for a BBE. Can you please check this and fix lines where we have more than 100 characters in a line? (This only applies for the code content; including the comments).